codegen/lib/templates/java/header.erb

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2017-2021 Trust Wallet.
+// Copyright © 2017-2022 Trust Wallet.
```

Comment:
I think we can get current year from ruby code in a separate pr

Reply:
Good idea